docs/running-relayer.md

```diff
@@ -0,0 +1,345 @@
+# Running your own bridge relayer
+
+:warning: :construction: Please read the [Disclaimer](#disclaimer) section first :construction: :warning:
+
+## Disclaimer
+
+There are several things you should know before running your own relayer:
+
+- initial bridge version (we call it bridges v1) supports any number of relayers, but **there's no guaranteed
+compensation** for running a relayer and/or submitting valid bridge transactions. Most probably you'll end up
+losing tokens - please accept this fact;
+
+- even if your relayer has managed to submit a valid bridge transaction that has been included into the bridge
+hub block, there's no guarantee that you will be able to claim your compensation for that transaction. That's
+because compensations are paid from the account, controlled by relay chain governance and it can have no tokens
+to compensate your useful actions. We'll be working on a proper process to resupply it on-time, but we can't
+provide any guarantee until that process is well established.
+
+## A Brief Introduction into Relayers and our Compensations Scheme
+
+Omitting details, relayer is an offchain process that is connected to both bridged chains. It looks at the
+outbound bridge messages queue and submits message delivery transactions to the target chain. There's a lot
+of details behind that simple phrase - you could find more info in the
+[High-Level Bridge Overview](./high-level-overview.md) document.
+
+Reward that is paid to relayer has two parts. The first part static and is controlled by the governance.
+It is rather small initially - e.g. you need to deliver `10_000` Kusama -> Polkadot messages to gain single
+KSM token.
+
+The other reward part is dynamic. So to deliver an XCM message from one BridgeHub to another, we'll need to
+deliver several "things" from one chain to another (omitting details: this "thing" is an XCM message). Those
+"things" are submitted as transactions and every transaction has its cost, which is:
```

Comment:
Nit: To me it seems a bit hard to follow. What do you mean by `we'll need to
deliver several "things" from one chain to another` ?

Reply:
The idea was to avoid unrelated details in this document - i.e. no mentions of storage and finality proofs, delivery and confirmation transactions, ... For that there's another document. But I agree - "things" aren't much better. I've removed this "things" "thing" and only left transactions

Reply:
Looks good ! The only thing left is that `deliver submit` seems a bit redundant.